### PR TITLE
add typescript definitions

### PR DESCRIPTION
**Pull request by georgyfarniev**
Hello. First of all, thank you for creating node-latex! Here is a typescript definitions for convenient usage in typescript projects.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * Compiles latex to pdf document
+ *
+ * @param doc
+ * The (La)TeX document you want to use.
+ *
+ * @param options
+ * The compiler options
+ */
+declare function latex(doc: Readable | string, options?: INodeLatexOptions)
+  : Readable
```
Comment (saadq) [inline index.d.ts]:
```suggestion
  : Transform
```

The only thing that needs to be fixed here is that `latex()` returns a `Transform` stream

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Readable } from 'stream'
```
Comment (saadq) [inline index.d.ts]:
```suggestion
import { Readable, Transform } from 'stream'
```

I believe you can just import `Transform` from here.

**Review by saadq — CHANGES_REQUESTED** [review]
This looks great, thanks very much for the PR @georgyfarniev! I was planning on doing a rewrite in TypeScript soon, but it would be good to have a libdef file in the mean time.

Just a small thing needs to be fixed, otherwise looks good!

**Comment by georgyfarniev** [comment]
@saadq done

**Review by saadq — APPROVED** [review]

**Comment by saadq** [comment]
Looks good, just published v`2.6.0` with the TypeScript libdef. Thanks again @georgyfarniev!

**Pull request merged**